examples/ui-demo/src/components/shared/mint-card/MintCardActionButtons.tsx

```diff
@@ -0,0 +1,54 @@
+import { ExternalLinkIcon } from "@/components/icons/external-link";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ExternalLinkIcon' is defined but never used.
<!-- __reviewdog__:ChA5MzFmZWE0ZjcyNjlmN2NlEgZyZGpzb24= -->